tests/test.env.example

```diff
@@ -1,3 +1,3 @@
-export KB_DEPLOYMENT_CONFIG_DIR=/Users/etc/etc/staging_service/deployment/conf
+export KB_DEPLOYMENT_CONFIG_DIR=/Users/<username>/<some_dir>/staging_service/deployment/conf
```

Comment:
Is this used in the tests anywhere? I'd assume something would need to change based on this change, but I'm not seeing it

Reply:
Yea there's no documentation for running these tests. I use this for running the tests in PYCHARM

Reply:
Resolving in favor of https://github.com/kbase/staging_service/issues/96